src/Core/Constants.php

```diff
@@ -202,6 +85,23 @@
     }
     define('BASE_PATH', $candidateBasePath);
 }
+
+// Allow a first class env var to be set that disables .env file loading
+if (!getenv('SS_IGNORE_DOT_ENV')) {
+    foreach (array(
+                 BASE_PATH,
+                 dirname(BASE_PATH),
+             ) as $path) {
+        try {
```

Comment:
Not a blocker, but I feel that an `if(file_exists())` check would be better than throwing and swallowing an exception.